Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/pending/SubmissionTestResultPendingViewModel.kt

```diff
@@ -32,6 +32,7 @@ class SubmissionTestResultPendingViewModel @AssistedInject constructor(
 
     init {
         Timber.v("init() coronaTestType=%s", testType)
+        updateTestResult()
```

Comment:
How is it fixing update in `HomeScreen` ? this is the pending test result screen right?

Reply:
From my understanding, this is not about the home screen but rather about test details.
I updated the description to be more clear.